pkg/loader/executor.go

```diff
@@ -102,17 +109,43 @@ func (tx *tx) commit() error {
 	return errors.Trace(err)
 }
 
+func (e *executor) updateMark(status int, channel string, tx *tx) error {
+	if e.info == nil {
+		return nil
+	}
+	values := map[string]interface{}{
+		loopbacksync.ChannelID:   e.info.ChannelID,
+		loopbacksync.Val:         status,
+		loopbacksync.ChannelInfo: channel}
+	columns := []string{loopbacksync.ChannelID, loopbacksync.Val, loopbacksync.ChannelInfo}
+	sql, args := updateMarkSQL(columns, values)
```

Comment:
Since the table definition is static, we may not need an extra function for dynamic creation here?
For example, we can just get the `SQL` and `args` ready in this function without using a `map[string]interface{}`.